### PR TITLE
Fix Jenkins Jobs Adding and Fixing Cleanup

### LINKED ISSUE #452: Fix Jenkins Jobs to Get Cleanup Working
### What You Are Seeing?

Some of the Jenkins scripts don't have a cleanup mechanism or if they don;t aren't being invoked correctly.

### What is Expected?
These jobs should cleanup the C:\windows\temp folder of any work being done as the job ends.

### How Did You Get This To Happen? (Steps to Reproduce)

Ran the Jenkins jobs and had leftover GUID folders with downloaded packages in C:\Windows\Temp

### PR DESCRIPTION
**Pull request by ryanrichter94**
## Description Of Changes
- Fix and add logic for cleanup in Jenkins scripts
- General page maintenance including linting, workspace dictionary adds, and styling changes

## Motivation and Context
Update page and make sure scripts are in line with QSG offering.

## Testing

* [X] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [X] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

Fixes #452

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,50 +1,61 @@
...
-    "MSIX",
-    "Ninite",
-    "SCCM",
-    "Statiq",
-    "TLDR",
-    "Uninstaller",
-    "Uninstalling",
+    "baretail",
     "baselined",
     "behavior",
```
Comment (pauby) [inline .vscode/settings.json:8]: If you're going to add this in can you also add in the other spelling of this word?

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,50 +1,61 @@
...
     "gemcutter",
     "gitextensions",
+    "Gitter",
     "honor",
+    "Internalizer",
+    "Kickstarter",
     "labeouf",
+    "LASTEXITCODE",
     "malware",
     "minecraft",
```
Comment (pauby) [inline .vscode/settings.json:33]: ?

File: .vscode/settings.json [hunk 3]
```diff
@@ -1,50 +1,61 @@
...
     "programdata",
+    "psobject",
     "rebundle",
+    "SCCM",
     "scriptable",
     "shia",
+    "Sonatype",
     "sphynx",
+    "Statiq",
     "sysadmins",
```
Comment (pauby) [inline .vscode/settings.json:54]:
```suggestion
    "SysAdmins",
```

File: .vscode/settings.json [hunk 4]
```diff
@@ -1,50 +1,61 @@
...
+    "TLDR",
     "uninstallation",
+    "Uninstaller",
+    "Uninstalling",
     "uninstalls",
     "versioningupgrades",
```
Comment (pauby) [inline .vscode/settings.json:60]: Is this corect?

**Comment by steviecoaster** [comment]
@pauby, you mention wanting him to rebase to see the settings.json changes, but they _are_ here, in two separate commits. I've helped him rebase his branch with master, but are you asking for this to be put into a _single_ commit on this PR? If that is the case, I can help me to do that work in addition to the rebase that we have just completed.

**Comment by ryanrichter94** [comment]
Hey @pauby, We have a plan to replace anywhere the Chocolatey Server setup was mentioned with a Nexus Setup Guide down the road. Mainly raised this issue with the context of updating the existing scripts with working removal code and bringing the current doc up to linting and styling standard.

**Comment by pauby** [comment]
The changes that are being made to raw URL's are not needed. Statiq renders them fine without the changes. Can these be removed?

**Review by pauby — APPROVED** [review]

**Comment by pauby** [comment]
@ryanrichter94 awesome work! Thanks for persevering and updating these docs!

**Pull request merged**